kong/plugins/oidc/handler.lua

```diff
@@ -29,27 +29,52 @@ function CustomHandler:access(config)
 
     session.configure(config)
 
-    local res, err = require("resty.openidc").authenticate(oidcConfig)
+    if tryIntrospect(oidcConfig) then
```

Comment:
Please, update also README.md and explain "two legged authentication" usage maybe at "What is Kong OIDC plugin" section.